week 1/lists_n_loops/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""Python lists can be iterated through different means, which include but not limited to:
+
+-> simple for loop
+-> for loop with range
+-> while loop
+-> enumerate
+-> list comprehension
+Each of these methods does the same thing. But are there any performance differences with each of these methods? 
+
+The purpose of this task is to:
+
+-> Practice different loop iteration methods
+-> Perform a time execution analysis for a list against all of these methods
+-> Plot the results
+-> Find out if one method is superior in terms of performance and why
+
+Script requirements:
+
+-> The list size should be variable, provided either via command-line argument or user input
+-> Some operations should be performed on each element of the list. That operation can be mutable/immutable.
+-> Don't print the items of the list during iteration. That can impact the execution time.
+-> Use pep8 conventions when writing the code.
+"""
```

Comment:
make this readme.md instead of adding in init.py

Reply:
Edited